src/header/Header.scss

```diff
@@ -14,16 +65,20 @@ header {
 
     &:hover {
       :global {
-        #bg{
+        #bg {
           fill-opacity: 0.35;
         }
       }
     }
   }
 }
 
-section {
+.section {
   display: inline-flex;
+
+  &Navigation {
+    margin-right: auto;
+  }
 }
 
 nav ul {
```

Comment:
and here too: add some classname to `ul` and use this classname here instead of `nav ul`